### PR TITLE
add styling

### PR DESCRIPTION
**Pull request by mortaen**
add a first styling

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,104 +6,112 @@
...
-            <li>
-                <a href="#contact"> Contact</a>
-            </li>
-            <li>
-                <a href="#bio"> Biography</a>
-            </li>
-        </ul>
-    </nav>
+        <nav class="navbar">
+            <h1 class="title">My Personal Website</h1>
```
Comment (macfrei) [inline index.html:15]: Usually the `h1` is not in the `nav` element as it is not a link or anything.

File: index.html [hunk 2]
```diff
@@ -6,104 +6,112 @@
...
+            <div class="nav-links">
+                <div class="nav-item"> Morten </div>
```
Comment (macfrei) [inline index.html:17]: This is also a bit confusing as it is not a `nav-item` per se. 

> The <nav> HTML element represents a section of a page whose purpose is to provide navigation links, either within the current document or to other documents. Common examples of navigation sections are menus, tables of contents, and indexes.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav

File: index.html [hunk 3]
```diff
@@ -6,104 +6,112 @@
...
-<main>
-
-    <section>
-        <img src="img/head-outline.png" alt="head outline">
-        <div>A placeholder photo.</div>
-    </section>
+    <main>
+        <br>
+        <br>
+        <br>
```
Comment (macfrei) [inline index.html:26]: You don't need to add these `br` elements. This should be worked on in CSS, for example with some margin.

File: index.html [hunk 4]
```diff
@@ -6,104 +6,112 @@
...
+        <section class="head-photo">
+            <img src="img/head-outline.png" alt="head outline">
+            <div>
+                A placeholder photo.
+            </div>
+        </section>
```
Comment (macfrei) [inline index.html:32]: Have a look at the `figure` element with `figcaption`.

File: index.html [hunk 5]
```diff
@@ -6,104 +6,112 @@
...
+        <section class="int-skill-section">
+            <h2 class="subheader">
+                Interests
+            </h2>
+            <div class="list-box">
+                <ul class=" list">
+                    <li class="list-item">football</li>
+                    <li class="list-item">skateboarding</li>
+                </ul>
+            </div>
```
Comment (macfrei) [inline index.html:43]: The div is probably not needed as you could give those styles the list directly. (usually that's the case 😉 )

File: index.html [hunk 6]
```diff
@@ -6,104 +6,112 @@
...
+
+                <tr>
+                    <td>email</td>
+                    <td>email@email.com</td>
+                </tr>
+                <tr>
+                    <td>phone</td>
+                    <td>111111111111</td>
+                </tr>
+            </table>
```
Comment (macfrei) [inline index.html:77]: Have a look at the `address` element and how to add phone numbers and emails to HTML

**Review by macfrei — COMMENTED** [review]
Good job! Added a few comments you might want to consider 🙂 👍

File: index.html (hunk 3 above)
Comment (mortaen) [reply index.html]: Thanks, done now :)

File: index.html (hunk 2 above)
Comment (mortaen) [reply index.html]: Tried to incorporate that, hopefully it looks better now.

**Pull request merged**